### PR TITLE
Translation tweaks.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
Incorporating Jonny's feedback.

### CODE REVIEW

File: lang/fr.json [hunk 1]
```diff
@@ -390,7 +390,11 @@
...
   },
-  "load-error-makecode": {
+  "load-error-makecode-info": {
+    "defaultMessage": "This hex file cannot be loaded in the Python Editor. The Python Editor cannot open hex files created with Microsoft MakeCode.",
+    "description": "Load error message"
+  },
+  "load-error-makecode-link": {
     "defaultMessage": "This hex file cannot be loaded in the Python Editor. The Python Editor cannot open hex files created with Microsoft MakeCode.",
     "description": "Load error message"
   },
```
Comment (microbit-robert) [inline lang/fr.json:400]: This will look a bit daft for French.

**Comment by microbit-robert** [comment]
To fix tests, change out "Downloaded 1 of 2 files" for "Warning: Only main.py downloaded" in `open.test.ts` and `save.test.ts`.

File: lang/fr.json (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply lang/fr.json:400]: Oops, will fix.

**Review by microbit-robert — APPROVED** [review]
All looks good. Suggest manually updating the MakeCode hex error message for French so it's not the same message duplicated. e2e fixes are straightforward.

File: lang/fr.json (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply lang/fr.json]: bdeddd1138db97dd701a3f6b739d32a4cd6ec365

**Comment by microbit-matt-hillsdon** [comment]
> All looks good. Suggest manually updating the MakeCode hex error message for French so it's not the same message duplicated. e2e fixes are straightforward.

In bdeddd1138db97dd701a3f6b739d32a4cd6ec365

**Pull request merged**